test/controllers/podcasts_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class PodcastsControllerTest < ActionController::TestCase
+  setup do
+    @podcast = podcasts(:one)
+  end
+
+  test 'should get index' do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:podcasts)
+  end
+
+  test 'should get new' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.